reflect.go

```diff
@@ -122,6 +122,9 @@ type Reflector struct {
 	// switching to just allowing additional properties instead.
 	IgnoredTypes []any
 
+	// Ignore specific struct field, enable dynamic generate different schemas from on struct.
+	Ignore func(reflect.StructField) bool
```

Comment:
Its not clear how this should be used... as a rule, I'd try to avoid using reflect types directly.